examples/resources/vpcs.tf

```diff
@@ -0,0 +1,27 @@
+terraform {
```

Comment:
slight filepath change to get this on the registry:

```
examples/resources/timescale_vpcs/resource.tf
```

[reference docs](https://github.com/hashicorp/terraform-provider-scaffolding-framework/tree/main/examples)